lib/elixir/lib/enum.ex

```diff
@@ -1493,6 +1493,9 @@ defmodule Enum do
       iex> Enum.into([a: 1, a: 2], %{})
       %{a: 2}
 
+      iex> Enum.into([a: 2], %{a: 1})
+      %{a: 2}
```

Comment:
Maybe we can have another key in there too, to show "merging":

```suggestion
      iex> Enum.into([a: 2], %{a: 1, b: 3})
      %{a: 2, b: 3}
```